app/assets/javascripts/components/jquery_wrappers/TaggedInput.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 
 class TaggedInput extends React.Component {
+  constructor() {
```

Comment:
I don't think we need to do that once we aren't using any prop in the constructor function.